src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -672,6 +685,30 @@ export async function activateRoslynLanguageServer(
 
     // Start the language server.
     _languageServer.start();
+
+    return _languageServer;
+
+    function scanExtensionPlugins(): string[] {
+        return vscode.extensions.all.flatMap((extension) => {
+            const loadPaths = extension.packageJSON.contributes?.['csharpExtensionLoadPaths'];
+            if (loadPaths === undefined || loadPaths === null) {
+                _traceChannel.appendLine(`Extension ${extension.id} does not contribute csharpExtensionLoadPaths`);
+                return [];
+            }
+
+            if (!Array.isArray(loadPaths) || loadPaths.some((loadPath) => typeof loadPath !== 'string')) {
+                _channel.appendLine(
+                    `Extension ${
+                        extension.id
+                    } has invalid csharpExtensionLoadPaths. Expected string array, found ${typeof loadPaths}`
+                );
+                return [];
+            }
+
+            _traceChannel.appendLine(`Extension ${extension.id} contributes csharpExtensionLoadPaths: ${loadPaths}`);
+            return loadPaths;
```

Comment:
We don't have to join the path with the extension path to create a full path?

Reply:
Good point, thanks.